api/applications/applications.proto

```diff
@@ -0,0 +1,28 @@
+
+syntax = "proto3";
+
+package wego_server.v1;
+
+option go_package = "github.com/weaveworks/weave-gitops/pkg/protos/applications";
+
+import "google/api/annotations.proto";
```

Comment:
It'd be nice to bring in the openapi annotations...how about something like...

```protobuf
import "protoc-gen-openapiv2/options/annotations.proto";

option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
  info: {
    title: "WeGo Applications API",
    version: "0.1";
    description: "<insert something here>";
  };
  consumes: "application/json";
  produces: "application/json";
};
```

This gets output in the resulting swagger.json

Reply:
Added